app/js/ui/SimpleModal.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { autobind } from './decorators';
+
+export default class SimpleModal extends React.Component {
+    componentDidMount() {
+        $(this.refs.modal.getDOMNode())
+            .openModal(this._modalOpts);
+    }
+
+    render() {
+        return (
+            <div ref="modal" className="modal modal-fixed-footer">
+                <div className="modal-content">
+                    <h2>{this.props.title}</h2>
+                    <p>{this.props.message}</p>
+                </div>
+                <div className="modal-footer">
+                    <a href="#!" className="modal-action modal-close waves-effect waves-green btn-flat ">Ok</a>
+                </div>
+            </div>
+        );
+    }
+
+    get _modalOpts() {
+        return { complete: this._handleModalComplete };
+    }
+
+    @autobind
+    _handleModalComplete() {
+        if (typeof this.props.onComplete === "function") {
```

Comment:
I wonder if there's a cool way to re-use this pattern other places - would maybe make it more succinct as well.
:)